packages/artifacts/chunks/languages/library.ts.txt

```diff
@@ -9,6 +9,7 @@
 
 import {DynamicNode} from "@lionweb/core";
 
+
```

Comment:
Two empty lines? Is this idiomatic in TS?

Reply:
Not really, but I tend to use extra whitespace between sections of code.